docker-compose.yml

```diff
@@ -60,6 +60,8 @@ services:
       - traefik.http.routers.appwrite_api_https.rule=PathPrefix(`/`)
       - traefik.http.routers.appwrite_api_https.service=appwrite_api
       - traefik.http.routers.appwrite_api_https.tls=true
+      # live reloader
+      - appwrite-dev-reload
```

Comment:
Any reason why we shouldn't have this on all Appwrite containers (that mount `app/` and `src/`)?

Reply:
Ehm I kind of forgot them since I haven't personally had the need to reload them, but I'll add it to the container that have those mounted.